handlers/officerDeltaHandler.go

```diff
@@ -1,22 +1,43 @@
 package handlers
 
 import (
+	"github.com/companieshouse/chs-delta-api/helpers"
+	"github.com/companieshouse/chs-delta-api/services"
 	"net/http"
 )
 
-// OfficerDeltaHandler offers a handler by which to publish a message onto a kafka topic.
+const (
+	OfficersTopic = "officers-delta"
```

Comment:
We should move the topic name to configs.

Reply:
This has been addressed in a future ticket and the low level design. We aim to do this but not for MVP